### PR TITLE
zm - make google search results all nicely formatted cards

### LINKED ISSUE #127: Search Results on the screen https://proj-mapache-search.herokuapp.com/member/search should be formatted nicely, not just dumped as JSON.
# User Story

As a member, when I do a Google Search from the page <https://proj-mapache-search.herokuapp.com/member/search>, I can see nicely formatted search results

# Discussion

Here's is how search results are formatted now:

image

Instead, we should see nice cards for each search result from the `items` array.

The card should have the `title` field (though probably use the `htmlTitle` field to display), the `htmlSnippet`, and the `htmlFormattedUrl` should link to the actual `url`.

**Example of what these "cards" looked like for [mapache search w20 edition](http://mapache-search.herokuapp.com/):**
Not saying it has to look exactly like this, also  you can see that there are some formatting issues, but the cards should  look similar to this. You can change it though up to your UI/UX discretion. Just think of how you want google search results to look like on this page.
[Code for the w20 edition here if you want to refer to it.](https://github.com/ucsb-cs56-w20/mapache-search)
image

### PR DESCRIPTION
**Pull request by ZechenM**
In this pr, we make modifications on the results of google search with transforming the json formatted results to nicely formatted cards. closes #127

### CODE REVIEW

**Review by chengyanwu — DISMISSED** [review]
LGTM

**Comment by ZechenM** [comment]
Passed all checks and code coverage

**Review by alonkatzcs — APPROVED** [review]
LGTM

File: javascript/src/test/utils/fetch.test.js [hunk 1]
```diff
@@ -13,6 +13,11 @@ describe("fetch tests", () => {
     });
     getToken.mockResolvedValue("token");
   });
+
```
Comment (zminshall) [inline javascript/src/test/utils/fetch.test.js:16]: Is this dummy test necessary?

File: javascript/src/test/utils/fetch.test.js [hunk 2]
```diff
@@ -13,6 +13,11 @@ describe("fetch tests", () => {
...
+  test("dummy test", async () => {
+  
+    expect({}).toEqual({});
+  });
```
Comment (zminshall) [inline javascript/src/test/utils/fetch.test.js:20]: Is this dummy test necessary?

Comment (ZechenM) [reply javascript/src/test/utils/fetch.test.js:20]: Probably not - let me update it

Comment (ZechenM) [reply javascript/src/test/utils/fetch.test.js]: should be good now

**Review by alonkatzcs — APPROVED** [review]
LGTM

**Review by jasunchen — APPROVED** [review]
lgtm

**Review by zminshall — APPROVED** [review]
Looks good!

**Pull request merged**